src/overview/mining/PrismDisplay.tsx

```diff
@@ -0,0 +1,64 @@
+import IPimg from "../../util/IPimg";
+import { useNumberItemObserver } from "../setItems/useSetItemsObserver";
+import { MouseEvent } from "react";
+import { sendMessage } from "../../util/websocket/useWebsocket";
+import { useTooltip } from "../../util/tooltip/useTooltip";
+import Tooltip from "../../util/tooltip/Tooltip";
+
+interface Props {
+  prism: string;
+}
+
+const PrismDisplay = ({ prism }: Props) => {
+  const [amount, setAmount] = useNumberItemObserver(
+    prism + "_stardust_prism",
+    `PrismDisplay-${prism}`
+  );
+
+  const onPrismClick = (event: MouseEvent) => {
+    hideTooltip();
+    if (event.shiftKey) {
+      setAmount(amount - 1);
+      sendMessage("SMASH_STARDUST_PRISM", prism + "_stardust_prism", amount - 1);
+    } else {
+      setAmount(amount);
+      sendMessage("SMASH_STARDUST_PRISM", prism + "_stardust_prism", amount);
+    }
+  };
+
+  const [prismProps, PrismToolTip, hideTooltip] = useTooltip(
+    [
+      <Tooltip
+        text={`Smash ${amount} ` + Items.get_pretty_item_name(prism) + ` Prism(s)`}
```

Comment:
Could we just figure out whether it should be plural or not, instead of writing "(s)"? something like
```ts
const pluralMarker = (amount: number): string => amount > 1 ? 's' : ''

text = {`Smash ${amount} ${Items.get_pretty_item_name(prism)} Prism${pluralMarker(amount}`}
```


This also applies to minerals, geodes, and moonstone